docs/deployment-guide.md

```diff
@@ -257,6 +259,10 @@ Note, if you do not set the property `sleeper.retain.infra.after.destroy` to fal
 choose to tear down Sleeper later on you will also need to destroy some further S3 buckets and DynamoDB tables manually.
 This is because by default they are kept.
 
+Note, if you do not set the property `sleeper.retain.logs.after.destroy` to false when deploying then however you
+choose to tear down Sleeper later on you will also need to delete some further log groups manually.
+This is because by default they are kept.
```

Comment:
It doesn't seem like this is something someone would normally do, if this property is set correctly. I guess the point we're trying to make here is to explain the default behaviour. Can we combine this into the paragraph before somehow?